api/presenter/service_binding.go

```diff
@@ -41,6 +41,12 @@ type ServiceBindingLinks struct {
 	Details         Link `json:"details"`
 }
 
+type ServiceBindingDetailsResponse struct {
+	Credentials    map[string]any `json:"credentials"`
+	SyslogDrainURL *string        `json:"syslog_drain_url,omitempty"`
```

Comment:
Similar to the `ServiceBindingDetailsRecord`, we do not support `SyslogDrainURL` and `VolumeMounts`, so I would recommend removing them from the response type.